docs/projects/adrv9026/index.rst

```diff
@@ -0,0 +1,441 @@
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :dokuwiki:`[Wiki] ADRV9026 & ADRV9029 Prototyping Platform User Guide <resources/eval/user-guides/adrv9025>`
+
+Here you can find the quick start guides available for these evaluation boards:
+
+.. list-table::
+   :widths: 20 10
+   :header-rows: 1
+
+   * - Evaluation board
+     - Zynq UltraScale+ MP
+   * - ADRV9026/ADRV9029
+     - :dokuwiki:`ZCU102 <resources/eval/user-guides/adrv9026/quickstart/zynqmp>`
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheets:
+
+   -  :adi:`ADRV9026`
+-  `UG-1578, Device User Guide <https://www.analog.com/media/radioverse-adrv9026/adrv9026-system-development-user-guide-ug-1727.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`ADRV9026 HDL project source code <projects/adrv9026>`
+
+.. list-table::
+   :widths: 30 40 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac`
+     - :ref:`here <axi_dmac>`
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - UTIL_CPACK2
+     - :git-hdl:`library/util_pack/util_cpack2`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/util_cpack>`
+   * - UTIL_UPACK2
+     - :git-hdl:`library/util_pack/util_upack2`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/util_upack>`
+   * - UTIL_ADXCVR for AMD
+     - :git-hdl:`library/xilinx/util_adxcvr`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/util_xcvr>`
+   * - AXI_ADXCVR for Intel
+     - :git-hdl:`library/intel/axi_adxcvr`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_adxcvr>`
+   * - AXI_ADXCVR for AMD
+     - :git-hdl:`library/xilinx/axi_adxcvr`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_adxcvr>`
+   * - AXI_JESD204_RX
+     - :git-hdl:`library/jesd204/axi_jesd204_rx`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/jesd204/axi_jesd204_rx>`
+   * - AXI_JESD204_TX
+     - :git-hdl:`library/jesd204/axi_jesd204_tx`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/jesd204/axi_jesd204_tx>`
+   * - JESD204_TPL_ADC
+     - :git-hdl:`library/jesd204/ad_ip_jesd204_tpl_adc`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/jesd204/jesd204_tpl_adc>`
+   * - JESD204_TPL_DAC
+     - :git-hdl:`library/jesd204/ad_ip_jesd204_tpl_dac`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/jesd204/jesd204_tpl_dac>`
+
+-  :dokuwiki:`[Wiki] Generic JESD204B block designs <resources/fpga/docs/hdl/generic_jesd_bds>`
+-  :dokuwiki:`[Wiki] JESD204B High-Speed Serial Interface Support <resources/fpga/peripherals/jesd204>`
```

Comment:
Please replace this line like this: (I created a PR to update the links to JESD documentation, since now we have this as well posted on GitHub)
```suggestion
-  :ref:`jesd204`
```